src/install/python/python-frameworks/python-agent-fastcgi.mdx

```diff
@@ -0,0 +1,40 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using FastCGI
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
```

Comment:
This section looks good. There might be a little repeat between the server and this section but I think that's ok for now.